packages/std/src/mock.rs

```diff
@@ -156,6 +156,11 @@ pub fn mock_info<U: Into<HumanAddr>>(sender: U, sent: &[Coin]) -> MessageInfo {
     }
 }
 
+// Returns mock env info as a tuple
+pub fn mock_env_info<U: Into<HumanAddr>>(sender: U, sent: &[Coin]) -> (Env, MessageInfo) {
```

Comment:
This is marked as unused because it is not exported outside of the crate (see `packages/std/src/lib.rs`)